README.markdown

```diff
@@ -31,7 +31,8 @@ This should give you the flexibility you need for customizing your input/output
 
 ### Building Your Phar
 
-Simply run `php package.php`, which will create the file: `build/doctrine-migrations.phar` for you.
+The PHAR is ready for [download here](http://github.com/downloads/ericclemmons/migrations/doctrine-migrations.phar), but if you still would like to build your PHAR
```

Comment:
and I'm not even sure why phars are provided for this package actually. I always use it integrated in my project, not as a standalone CLI tool